packages/paste-website/src/pages/components/checkbox/index.mdx

```diff
@@ -107,8 +111,7 @@ A checkbox is always displayed next to a visible label.
 
 ### Checkbox with help text
 
-In cases where the checkbox requires additional context, you can display this information as help text below the checkbox control and label. This can help keep checkbox labels concise. In order to maintain styling consistency, be sure to use the `helpText` prop here instead of using the Help Text component.
-
+Use Checkbox with Help Text in cases where the checkbox requires additional context. In order to maintain styling consistency, be sure to use the `helpText` prop here instead of using the Help Text component.
```

Comment:
```suggestion
Use Checkbox with help text in cases where the checkbox requires additional context. In order to maintain styling consistency, be sure to use the `helpText` prop here instead of using the Help Text component.
```